package.json

```diff
@@ -6,39 +6,51 @@
   "author": "Julian Venci",
   "license": "M.I.T.",
   "engines": {
-    "node": ">=6.13.0"
+    "node": "8.15.1"
   },
   "scripts": {
     "start": "nodemon ./server/index.js",
     "build": "webpack --watch",
```

Comment:
@Jvenci 
You would need to remove the '--watch' if you are using it with CircelCI as it will timeout. Probably create a "build" and "build:watch" would work

Reply:
Interesting, are you saying create two builds? One with watch and one without?

Reply:
nevermind, i see what you meant, thanks for the recommendation